template/.gitignore

```diff
@@ -3,3 +3,4 @@ fileloader.ts
 
 # Folder made by Kwik db to watch guild command versions
 db/
+node_modules
```

Comment:
next time try not to accidentally include unrelated changes! with that being said, I see nothing wrong with this, it's just that it shouldn't be in a PR named "Forum channels"